Documentation/ApiOverview/Services/Developer/ServiceApi.rst

```diff
@@ -7,15 +7,19 @@
 Service API
 ===========
 
-All service classes must inherit from the base service class
-:php:`\TYPO3\CMS\Core\Service\AbstractService`,
-unless the service type provides a specific
-base class (authentication services, for example, inherit from
-:php:`\TYPO3\CMS\Core\Authentication\AbstractAuthenticationService` instead).
-These specific classes should
-normally themselves extend :php:`\TYPO3\CMS\Core\Service\AbstractService`.
-This class provides a large number of important or useful methods which are
-described below, grouped by type of usage.
+.. deprecated:: 11.3
+   The abstract class :php:`\TYPO3\CMS\Core\Service\AbstractService` has been
+   deprecated. See :ref:`services-developer-service-api-migration`.
+
+.. important::
+   The Service API never found many usages in casual extensions. It is mainly
+   used for authentication services. It is planed to deprecate the complete
```

Comment:
I would remove this statement: we don't know for sure how it was used by extension developers and we are not pointing out how to do it differently - as a reader, I just don't know what to do with this info without more context.